demo_oc3_2024/evidence-endorsements.textproto

```diff
@@ -1,15 +1,20 @@
 oak_restricted_kernel {
```

Comment:
FWIW, consider including https://github.com/project-oak/oak/pull/4898 in your branch if you have problems. (But don't do it if you are happy with the demo overall.)

Reply:
Very cool! Not sure we have time to get new endorsements for our our binaries though